### PR TITLE
Identify ways in which Atomist can be incorporated into existing tooling

### PR DESCRIPTION
**Pull request by gjtorikian**

### CODE REVIEW

**Comment by gjtorikian** [comment]
I think it's safe to mark this ready for review too. Essentially I went about consolidating multiple pieces of content on Atomist integrating with other tools in one place. I tried to draw a clear distinction between what's natively supported (and where to find more tools) and point out how to integrate other tools when necessary.

File: docs/user/admin/integrations/supported-integrations.md [hunk 1]
```diff
@@ -31,29 +29,43 @@ include action buttons to create and merge PRs and add labels, reviewers, and co
...
+
+## Running other programs from Atomist
 
 In response to events, you can trigger actions on other systems from your Atomist automations.
-Some of the ones already implemented include:
+Some of the ones we've already implemented include:
 
-* Checkstyle
+* [Checkstyle](/pack/checkstyle/)
 * TSLint
```
Comment (gjtorikian) [inline docs/user/admin/integrations/supported-integrations.md]: I couldn't find a link to this either as a package or a repo, so I hope leaving this blank is ok!

File: docs/user/admin/integrations/supported-integrations.md [hunk 2]
```diff
@@ -31,29 +29,43 @@ include action buttons to create and merge PRs and add labels, reviewers, and co
...
-* Sonarqube
-* anything you can run from a shell or command prompt
+* [Sonarqube](/pack/sonarqube/)
+
+You can find a full list of the packages we support and maintain [in our GitHub org](https://github.com/atomist?utf8=%E2%9C%93&q=sdm-pack-&type=&language=).
```
Comment (gjtorikian) [inline docs/user/admin/integrations/supported-integrations.md]: Can it be expected that every repo starts with `sdm-pack-`?

Comment (jessitron) [reply docs/user/admin/integrations/supported-integrations.md]: Yes

**Comment by jessitron** [comment]
“Atomist isn't designed to replace your CI service;”

@jrday is this the latest? Because it isn’t what I’ve been saying. Atomist does not have to replace CI but it can?

**Comment by jessitron** [comment]
Can we avoid the words “simple” and “simply”? This stuff is not simple when you don’t already know how to do it.

File: docs/user/admin/integrations/supported-integrations.md [hunk 3]
```diff
@@ -1,26 +1,24 @@
-
-Atomist receives events from and performs operations on many systems,
+Atomist isn't designed to replace your CI service; rather, the goal of the SDM is to enhance all of your existing pipelines and policies through additional automation. It does so by receives and reacting to events from many systems,
```
Comment (jrday) [inline docs/user/admin/integrations/supported-integrations.md]: > ~Atomist isn't designed to replace your CI service; rather,~
I suggest striking that phrase. The underlying idea is that you don't have to replace your CI in order to use Atomist.

**Comment by gjtorikian** [comment]
> Can we avoid the words “simple” and “simply”? This stuff is not simple when you don’t already know how to do it.

This was a straight copy of [the existing text](https://github.com/atomist/docs/pull/326/files#diff-cacc5a8626a2cc71344704d473c67b79L157), but I definitely agree and will remove it.

**Pull request merged**